package.json

```diff
@@ -7,47 +7,28 @@
     "email": "o.o@mug.dog",
     "url": "http://dafrok.github.io"
   },
-  "contributors": [
-    {
-      "name": "Zdravko Ćurić",
-      "url": "https://github.com/zcuric"
-    }
-  ],
-  "homepage": "https://github.com/Dafrok/v-hotkey#readme",
-  "bugs": {
-    "url": "https://github.com/Dafrok/v-hotkey/issues"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Dafrok/v-hotkey.git"
-  },
-  "keywords": [
-    "vue",
-    "hotkey",
-    "hotkeys",
-    "directive",
-    "shortcut",
-    "shortcuts"
-  ],
-  "license": "MIT",
   "scripts": {
+    "build": "vue-cli-service build --target lib --name v-hotkey ./src/index.js",
```

Comment:
I found these are just changes in order. What's the purpose of doing this?

Reply:
Rearrengement happened as side effect running vue cli inside the project but important change here, somehow hidden from plain sight 🙁 , is that `@vue/cli-plugin-babel` is being added to dev dependencies. You can safely ignore all other ordering changes.

Reply:
That's OK